### DISCUSSION TITLE
PMM/movie/franchise is editing movie.nfo files (solved)

### DISCUSSION THREAD

**Discussion by laryan61** [meisnate12/Plex-Meta-Manager-Configs#413] [General]
In my config.yml file I'm using your Franchise Collection.

- git: PMM/movie/franchise

I had just recently went through my movie library, removing the <set> tags from all the movie.nfo files. Now I've discovered it looks like PMM has added them all back in. I'd prefer the NFO files to not be touched and PMM control what's in my collections. Is there a setting in Plex or PMM to not edit/update the NFO files with the collection <set> tags?

**Comment by laryan61** [comment]
After additional testing. I discovered it was Radarr that was periodically re-writing/re-creating the movie NFO files with the collection <set> tags. Which then Plex was using to create new movie collections. Because I was using the XBMC NFO metadata scraper in Plex, which scraped the NFO when it saw the change/update. I've since uninstalled/removed that meta scraper from Plex.

**Reply by cpt-kuesel** [reply]
Thanks for reporting back and glad you identified the culprit!
